xknx/knxip/cemi_frame.py

```diff
@@ -106,20 +107,25 @@ def calculated_length(self):
     def from_knx(self, raw):
         """Parse/deserialize from KNX/IP raw data."""
         try:
-            self.code = CEMIMessageCode(raw[0])
-        except ValueError:
-            raise CouldNotParseKNXIP("Could not understand CEMIMessageCode: {0} ".format(raw[0]))
-
-        if self.code == CEMIMessageCode.L_DATA_IND or \
-                self.code == CEMIMessageCode.L_Data_REQ or \
-                self.code == CEMIMessageCode.L_DATA_CON:
-            return self.from_knx_data_link_layer(raw)
-        raise CouldNotParseKNXIP("Could not understand CEMIMessageCode: {0} / {1}".format(self.code, raw[0]))
+            try:
+                self.code = CEMIMessageCode(raw[0])
+            except ValueError:
+                raise UnsupportedCEMIMessage("CEMIMessageCode not implemented: {0} ".format(raw[0]))
+
+            if self.code == CEMIMessageCode.L_DATA_IND or \
+                    self.code == CEMIMessageCode.L_Data_REQ or \
+                    self.code == CEMIMessageCode.L_DATA_CON:
+                return self.from_knx_data_link_layer(raw)
+            raise UnsupportedCEMIMessage("Could not handle CEMIMessageCode: {0} / {1}".format(self.code, raw[0]))
+        except UnsupportedCEMIMessage as unsupported_cemi_err:
+            self.xknx.logger.warning("Ignoring not implemented CEMI: %s", unsupported_cemi_err)
+            return len(raw)
```

Comment:
Do we need additional handling within the calling function? (bc no error is raised?)

Reply:
`UnsupportedCEMIMessage` doesn't leave the CemiFrame module. Its just used to get the log message. 
Let's see if we run into troubles this way 🍌